api/src/test/java/com/github/eyefloaters/console/api/support/EnabledOperationFilterTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.eyefloaters.console.api.support;
+
+import jakarta.ws.rs.core.Response.Status;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+
+import static com.github.eyefloaters.console.test.TestHelper.whenRequesting;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+
+@QuarkusTest
+@TestProfile(OperationFilterTestProfile.class)
+class EnabledOperationFilterTest {
+
+    @Test
+    void testDescribeTopicNotAllowed() throws Exception {
+        whenRequesting(req -> req.get("/api/kafkas/{clusterId}/topics/{topicId}", "kafka-1", "topic-1"))
```

Comment:
Is the clusterId really a name like that, or is it the real Kafka cluster id?

Reply:
For a working endpoint it would be a Kafka-formatted `Uuid`. The server handler never makes it past the filter where the request is requested with HTTP 405. I'll update the test with random Uuids to be less confusing.